Sources/SwiftGenKit/Parsers/Yaml/YamlParser.swift

```diff
@@ -33,7 +33,7 @@ public enum Yaml {
     }
 
     public class var defaultFilter: String {
-      "[^/]\\.(?i:ya?ml)$"
+      filterRegex(for: ["ya?ml"])
```

Comment:
After `filterRegEx(forExtensions:)` ends up applying `NSRegularExpression.escape(…)` (see comment above) then for this we would use `["yml", "yaml"]` instead – which would actually make it more readable imho